### PR TITLE
Extracts response mocks to a fixtures

### PR DESCRIPTION
**Pull request by waldemarnt**

### CODE REVIEW

**Comment by jeffhsta** [comment]
:+1:

File: test/server_spec.js [hunk 1]
```diff
@@ -92,60 +57,9 @@ describe('EquipmentController', () => {
...
-                      }
-                    ]
-                  }
-                ]
-              }
-            ]
-          }
-        }
-      }
+      let telemetryReponse = readFixture('telemetrySearch', {});
```
Comment (bltavares) [inline test/server_spec.js]: There is no need to pass the `{}` if there is nothing to merge. `Object.assign` will happily take undefined.

File: test/server_spec.js [hunk 2]
```diff
@@ -224,38 +138,10 @@ describe('EquipmentController', () => {
...
-                    ]
-                  }
-                ]
-              }
-            ]
-          }
-        }
-      }
+      
+      let telemetryResponse = readFixture('telemetrySearch', {});
```
Comment (bltavares) [inline test/server_spec.js]: Same here

File: test/fetcher/canVariablesFetcher_spec.js [hunk 3]
```diff
@@ -3,60 +3,13 @@ import CanVariablesFetcher from '../../fetcher/canVariablesFetcher.js';
...
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        }
-      }
-    }
+    let telemetryResponse = readFixture('telemetrySearch', {});
```
Comment (bltavares) [inline test/fetcher/canVariablesFetcher_spec.js]: Same here

**Comment by bltavares** [comment]
LGTM

**Comment by bltavares** [comment]
:ship: :it:

File: test/server_spec.js (hunk 1 above)
Comment (waldemarnt) [inline test/server_spec.js]: Really good observation, thanks!

**Comment by glauco** [comment]
LGTM :+1:

**Pull request merged**